src/chatproviderservice/src/Client/ChatProviderServiceClient.lua

```diff
@@ -53,6 +67,20 @@ function ChatProviderServiceClient:Start()
 	end
 end
 
+--[=[
+	Sends a system message to the provided TextChannel.
+	@param channel TextChannel
+	@param message string
+	@param color Color3?
+]=]
+function ChatProviderServiceClient:SendSystemMessage(channel: TextChannel, message: string, color: Color3?)
```

Comment:
Can we make the channel optional and default it to the system channel? I have no idea, but it might be worth simplifying this API further.

Reply:
resolved in [20071a8](https://github.com/Quenty/NevermoreEngine/pull/520/commits/20071a874fd7dbe733fdc116789fa8993ef0a405)